atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/StartupIndependenceEteTest.java

```diff
@@ -147,9 +156,28 @@ private static boolean canPerformTransaction() {
             return true;
         } catch (AtlasDbRemoteException e) {
             return false;
+        } catch (Exception e) {
+            if (exceptionIsRetryableAndContainsMessage(e, "is not initialized yet")) {
+                return false;
+            }
+            throw e;
         }
     }
 
+    private void assertNotInitializedExceptionIsThrownAndMappedCorrectly() {
+        try {
+            addTodo();
+            fail();
```

Comment:
A message could be added here (something like 'expected an exception to be thrown')